computeResourcesInstallSteps.go

```diff
@@ -6,17 +6,23 @@ import (
 	"fmt"
 )
 
+const (
+	ComputeResourceInstallStepStatusError ComputeResourceInstallStepStatus = "error"
+)
+
+type ComputeResourceInstallStepStatus string
```

Comment:
Please place `type` before constants. It's a common way to define enums in Golang.